generator/src/lib.rs

```diff
@@ -2379,6 +2379,11 @@ pub fn extract_native_types(registry: &vk_parse::Registry) -> (Vec<(String, Stri
         match ty.category.as_deref() {
             Some("include") => {
                 // `category="include"` lacking an `#include` directive are generally "irrelevant" system headers.
+                // using inlined format args in panic messages is not yet supported and will generate this warning:
+                // warning: panic message contains an unused formatting placeholder
+                // note: this message is not used as a format string when given without arguments, but will be in Rust 2021
+                // note: `#[warn(non_fmt_panics)]` on by default
+                #[allow(clippy::uninlined_format_args)]
```

Comment:
Could we fix this by updating the edition to 2021?

Reply:
Done.